src/components/Article.js

```diff
@@ -0,0 +1,20 @@
+import { useEffect, useState } from "react";
+import { useParams } from "react-router-dom";
+import { fetchArticleById } from "../api";
+import ArticleTitleCard from "./ArticleTitleCard";
+
+export default function Article(){
+    const[currentArticle,setCurrentArticle]= useState(
+        {        })
+    const {article_id}=useParams()
+    useEffect(()=>{
+        fetchArticleById(article_id).then(({article})=>{
+            setCurrentArticle(article)
+        })
+    },[currentArticle])
+    return (<div>
+        article :
+        <ArticleTitleCard {...currentArticle}/>
```

Comment:
why is spread needed here? Viewing an object should in no way mutate it

Reply:
passing props